go/libkb/pipeowner_windows.go

```diff
@@ -93,16 +95,16 @@ func IsPipeowner(log logger.Logger, name string) (bool, error) {
 	if err != nil {
 		return false, err
 	}
-	isEqual := windows.EqualSid(pipeSid, userSid)
-	if !isEqual {
+	isOwner = windows.EqualSid(pipeSid, userSid)
+	if !isOwner {
 		pipeAccount, pipeDomain, pipeAccType, pipeErr := pipeSid.LookupAccount("")
 		userAccount, userDomain, userAccType, userErr := userSid.LookupAccount("")
 		log.Debug("Pipe account: %s, %s, %v, %v", pipeAccount, pipeDomain, pipeAccType, pipeErr)
 		log.Debug("User account: %s, %s, %v, %v", userAccount, userDomain, userAccType, userErr)
 		// If the pipe is served by an admin, let local security policies control access
 		if pipeAccount == "Administrators" && pipeDomain == "BUILTIN" && pipeAccType == syscall.SidTypeAlias {
-			return true, nil
+			isOwner = true
 		}
 	}
-	return isEqual, nil
+	return
```

Comment:
But still do `return isOwner, nil`, since that's less error-prone as the code changes in the future.